CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 CHANGELOG
 =========
 
+7.1.0-beta.1
+------------
+
+* Added support for the new risk reasons outputs in minFraud Factors. The risk
```

Comment:
Does this changelog entry seem fine? One thought would be to use language specific names like we do for some other things, i.e. reference `riskScoreReasons` instead of risk reasons.

Reply:
I think this is fine. Maybe it would be be helpful to have a short sentence mentioning that it is available at `response.riskScoreReasons` or something.